indico/MaKaC/PDFinterface/conference.py

```diff
@@ -1761,10 +1761,16 @@ def __init__(self, conf, badgeTemplate, marginTop, marginBottom, marginLeft, mar
         self.__marginColumns = marginColumns
         self.__marginRows = marginRows
         if registrantList == 'all':
-            self.__registrantList = Registration.find(~Registration.is_deleted, Registration.event_id == conf.id).all()
+            self.__registrantList = (Registration
+                                     .find(~Registration.is_deleted, Registration.event_id == conf.id)
```

Comment:
Wouldn't `Registration.is_active` make more sense? I don't think you want badges for rejected/withdrawn registrations